les/client.go

```diff
@@ -263,9 +263,9 @@ func (s *LightEthereum) APIs() []rpc.API {
 			Service:   NewPrivateLightAPI(&s.lesCommons),
 			Public:    false,
 		}, {
-			Namespace: "lespay",
+			Namespace: "vfx",
```

Comment:
`vfc`?

Reply:
or `vflux`?

Reply:
I think `vflux` perhaps is better? It's a common name in client side `vfc` and server side `vfs`